### PR TITLE
Fix notification system on Ubuntu installation

### PR DESCRIPTION
**Pull request by m-1-k-3**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Bug fix - Notification system on Ubuntu
Closes https://github.com/e-m-b-a/emba/issues/284
Fixes also the chown issue seen here https://github.com/e-m-b-a/emba/issues/287#issuecomment-1206882276

* **What is the current behavior?** (You can also link to an open issue here)

See https://github.com/e-m-b-a/emba/issues/284
See https://github.com/e-m-b-a/emba/issues/287

### CODE REVIEW

**Comment by torabi12** [comment]
A few minutes ago I tested emba installation, but it dropped this:
kép

**Comment by m-1-k-3** [comment]
Code that was not touched. Could you please try again.

**Comment by torabi12** [comment]
I am going on holiday, I will try again a week later....

File: helpers/helpers_emba_print.sh [hunk 1]
```diff
@@ -739,7 +739,8 @@ print_notification(){
   local CURRENT=""
   CURRENT=$(<"$NOTIFICATION_LOCATION")
 
-  inotifywait -m -e modify "$NOTIFICATION_LOCATION" --format "%e" | while read -r EVENT; do
+  disable_strict_mode "$STRICT_MODE" 0
```
Comment (p4cx) [inline helpers/helpers_emba_print.sh:742]: I am missing a `enable_strict_mode "$STRICT_MODE" 0` here

**Review by p4cx — CHANGES_REQUESTED** [review]

File: helpers/helpers_emba_print.sh (hunk 1 above)
Comment (m-1-k-3) [reply helpers/helpers_emba_print.sh:742]: As this process get killed on EMBA exit it is not needed

Comment (p4cx) [reply helpers/helpers_emba_print.sh:742]: Oh, that's true. I didn't think of that 😄

**Review by p4cx — APPROVED** [review]

**Comment by p4cx** [comment]
Thank you for the fast fix!

**Comment by torabi12** [comment]
The error on the above screenshot was a temporary issue, I was not able to reproduce it. Something was wrong on my test system.

**Pull request merged**